### DISCUSSION TITLE
Build multiple nextjs app images with docker

### DISCUSSION THREAD

**Discussion by volnei** [vercel/turborepo#4262] [Help] [answered]
How to build multiple docker images (one for each app in apps)?

**Comment by JasperAlexander** [comment]
I don't get your question. Do you want to run multiple apps with Docker?

**Comment by anthonyshew** [comment, marked as answer]
We have documentation for [how to build Docker images](https://turbo.build/repo/docs/handbook/deploying-with-docker) using [`prune`](https://turbo.build/repo/docs/reference/command-line-reference/prune).

For each app you'd want to dockerize, you'd follow these steps.
